e2e/cypress/integration/integrations/outgoing_webhook/token_copy_icon_spec.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+describe('Integrations', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
+        // # Create test team and channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team.name;
+            testChannel = channel.display_name;
+        });
+    });
+
+    it('MM-T616 Copy icon for Outgoing Webhook token', () => {
+        // Visit the integrations > add page
+        cy.visit(`/${testTeam}/integrations/outgoing_webhooks/add`);
+
+        // * Assert that we are on the add page
+        cy.url().should('include', '/outgoing_webhooks/add');
+
+        // # Manually set up an outgoing web-hook
+        cy.get('#displayName').type('test');
+        cy.get('#channelSelect').select(testChannel);
+        cy.get('#triggerWords').type('trigger');
+        cy.get('#callbackUrls').type('https://mattermost.com');
+        cy.get('#saveWebhook').click();
```

Comment:
Please use [API queries from testing-library](https://testing-library.com/docs/dom-testing-library/api-queries) as much as possible. So, in this case, this could be,
```suggestion
        cy.findByText('Save').click();
```